src/js/stores/ConferenceStore.jsx

```diff
@@ -0,0 +1,57 @@
+import { EventEmitter } from 'events';
+import AuthService from 'util/AuthService';
+import dispatcher from 'dispatcher';
+
+const url = 'https://dev.calligre.com';
```

Comment:
yeah I updated the urls recently but there were open branches that didn't get the changes.